test/spec/provider/cloud-element-templates/ElementTemplates.spec.js

```diff
@@ -139,6 +139,74 @@ describe('provider/cloud-element-templates - ElementTemplates', function() {
 
   });
 
+  describe('getAll', function() {
+
+    it('should get all templates', inject(function(elementTemplates) {
+
+      // when
+      const templates = elementTemplates.getAll();
+
+      // then
+      expectTemplates(templates, [
```

Comment:
From the API perspective a "get all (but only latest)" would be beneficial, too. 

So maybe taking inspriration from [`Overlays#get`](https://github.com/bpmn-io/diagram-js/blob/develop/lib/features/overlays/Overlays.js#L165) with a more generic search option could work, too? Just throwing that in as a suggestion.

Reply:
I have to think about the API a bit more. I see that this would be the default use-case for Template choosers, but the only things I can come up with are
- separate functions `getAll` and `getLatest`
- additional argument to include all versions (or only latest, depending on what we decide to be the default)

Reply:
Did you also check https://bpmn-io.github.io/bpmn-js-connectors-extension/?aa=1? Given the limited space for discovery (and the focus on ease of use) I'm inclined to show latest there only.

Reply:
Same applies to the [element-template-chooser](https://github.com/bpmn-io/element-template-chooser) where I'm inclined to only show latest.

Reply:
At least in the demos, we don't make use of versioning.

When I think about it the other way: in what circumstances to we need older versions? I don't see a use-case outside of "old template is applied, update to latest version". Even there, we only need "what is the latest version" to compare it to the currently applied one.
Therefore, let's hide this behind a flag/additional attribute (`elementTemplates.getAll(id|element, allVersions)`).

Reply:
On second thought, it does not make sense for `getAll(id)`, which is our current use-case and would be breaking behavior. Therefore, It makes more sense to have `getLatest` alongside `getAll`, which also makes sense for `getLatest(id)`

Reply:
Implemented in [fbd6d5a](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/665/commits/fbd6d5aff3f875917b36339d0223c8fd3d34113b). Ready for review :)

Reply:
> Therefore, let's hide this behind a flag/additional attribute (elementTemplates.getAll(id|element, allVersions)).

Exactly. Could not have phrased that any better.